puppet/zulip_ops/files/nagios_plugins/zulip_zephyr_mirror/check_personal_zephyr_mirrors

```diff
@@ -9,6 +9,7 @@ mirrors when they receive the messages sent every minute by
 /etc/cron.d/test_zephyr_personal_mirrors
 """
 from __future__ import print_function
+from typing import Dict
```

Comment:
The puppet ones should be in an `if False`, since the typing module may not be installed for puppet.